client/hotrod-client/src/test/java/org/infinispan/client/hotrod/transcoding/DataFormatIndexedTest.java

```diff
@@ -1,22 +0,0 @@
-package org.infinispan.client.hotrod.transcoding;
-
-import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
-
-import org.infinispan.configuration.cache.ConfigurationBuilder;
-import org.testng.annotations.Test;
-
-/**
- * Tests for the Hot Rod client using multiple data formats with indexing enabled
- *
- * @since 9.3
- */
-@Test(groups = "functional", testName = "client.hotrod.transcoding.DataFormatIndexedTest")
-public class DataFormatIndexedTest extends DataFormatTest {
```

Comment:
Rather that deleting this test, the base DataFormatTest should declare an entity the has ```@Indexed```, besides the ComplexKey